spec/php-spec.coffee

```diff
@@ -820,11 +820,11 @@ describe 'PHP grammar', ->
     it 'tokenizes function names with characters other than letters or numbers', ->
       # The space between foo and bar is a nbsp (char 160/hex 0xA0), not an actual space (char 32/hex 0x20)
       # 0xA0 is between 0x7F and 0xFF, making it a valid PHP identifier
-      tokens = grammar.tokenizeLines "<?php\nfunction foo bar() {}"
```

Comment:
What happened here? This might be the reason for the tests failing

Reply:
Wow, thanks for catching that.  I copied this file from my previous PR after it didn't make it through the rebase.  I guess Github's interface converts nbsps to spaces?